certified-connectors/Baron Blue/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1492 @@
+                  "title": "Text to convert to integer"
+                }
+              },
+              "required": [
+                "convert"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/v1/convert/string/to/float/": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "converted": {
+                  "type": "number",
+                  "description": "The converted float from the text",
+                  "title": "Converted Float",
+                  "format": "float"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your request. Check the error message for more details.",
+                  "title": "Bad Request"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was a problem with your authentication. Try refreshing the connection or check if your subscription is still valid.",
+                  "title": "Unauthorized"
+                }
+              }
+            }
+          },
+          "500": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "detail": {
+                  "type": "string",
+                  "description": "There was an unexpected error. The development team was notified.",
+                  "title": "Unexpected error"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Convert Text to Float",
+        "description": "Converts the text (e.g. \" 123,33\")  to a float value (123.33)",
+        "operationId": "ConvertTextToFloat",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "convert": {
+                  "type": "string",
+                  "description": "Please insert a text (e.g., \"123,33\") with a float or integer value",
+                  "title": "Text to convert to float"
+                },
+                "separator": {
+                  "type": "string",
+                  "description": "What is the decimal separator? Can be \u201c.\u201d or \u201c,\u201d",
```

Comment:
@mtgomes some unicode symbols here too, please replace them with corresponding symbols (e.g. quotation marks, etc) or remove.

Reply:
@mtgomes Please remove the unique symbols from all the descriptions in the swagger file.